package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@evojam/redux-utils",
+  "version": "1.0.0",
+  "main": "./src",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": "https://github.com/evojam/redux-utils.git",
+  "homepage": "https://github.com/evojam/redux-utils.git#readme",
+  "bugs": {
+    "url": "https://github.com/evojam/redux-utils/issues"
+  },
+  "author": "Krzysztof Antecki <kantecki@evojam.com>",
+  "license": "MIT",
+  "peerDependencies": {
+    "typescript": ">=2.8"
+  },
+  "devDependencies": {
+    "@types/jest": "^23.3.2",
+    "jest": "^23.6.0",
+    "ts-jest": "^23.10.0",
+    "typescript": ">=2.8"
```

Comment:
So it will install 3.0 for me and I still can use there features of 3.0 and import it into project with 2.8.

Result? Compile error :P

Reply:
Final version is: `peer > 2.8` and `dev >=2.8  <3.0` because of our internal `angular-bootstrap` project which unfortunately cannot be used with TS 3 (angular 6 is not supporting it)

Reply:
Great :) that kind of setup is fine (as long as we are not using 3.0 features there) :)